.github/labeler-pr-triage.yml

```
@@ -664,6 +664,10 @@ service/route53domains:
   - 'aws/internal/service/route53domains/**/*'
   - '**/*_route53domains_*'
   - '**/route53domains_*'
+service/route53recoverycontrolconfig:
+  - 'aws/internal/service/route53recoverycontrolconfig/**/*'
+  - '**/*_route53_recovery_control_config_*'
+  - '**/route53_recovery_control_config_*'
```

We prefer the service part of the resource type name to be a single word (i.e. no `-` or `_` characters) - this will aid with future automation.

```suggestion
  - '**/*_route53recoverycontrolconfig_*'
  - '**/route53recoverycontrolconfig_*'
```

---

Sure. fixed it